README.template

```diff
@@ -139,7 +139,7 @@ It is possible to execute `*.sh`, `*.sql` and/or `*.php` custom file at the end
 ```
 \docker-init.d
 |- custom_script.sql
-|- custom_script.php
+|- custom_script.php  // You should create your own module instead of doing this.
```

Comment:
I think this comment is not relevant. I used it now to do a quick customization on initialization.